transformer/cloudfoundrytransformer.go

```diff
@@ -263,21 +299,43 @@ func (t *CloudFoundry) prioritizeAndAddEnvironmentVariables(cfApp collecttypes.C
 			flattenedEnvList := flattenVcapServiceVariables(valueStr)
 			for _, env := range flattenedEnvList {
 				envOrderMap[env.Name] = env
```

Comment:
Why not call normalizeForEnvironmentVariableName function directly here?

Reply:
done